homework/content.js

```diff
@@ -1,37 +1,40 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+(function(exports) {
+    var ContentManager = function() {
+        this._wrapper = null;
+    };
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+    ContentManager.prototype = {
+        start() {
+            this._wrapper = document.querySelector('#note-content-wrapper');
+            window.addEventListener('note-open', (function(event) {
```

Comment:
You should set `this` as the listener as the second example in the manual:

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Why_use_addEventListener